src/javascripts/components/hub/airportComponent.js

```diff
@@ -0,0 +1,52 @@
+const buildAirport = (airport) => {
+  let domString = '';
+  domString += `
+        <div class="fancy-card airport">
+        <div class="additional">
+          <div class="user-card">
+            <img width="450" src="${airport.imgUrl}" alt="image of ${airport.airportName}">
+          </div>
+          <div class="more-info shading" >
+            <h1 class="mb-3 mt-3 ml-3 mr-3" style="font-size:1.2rem">${airport.airportName}</h1>
```

Comment:
for accessability reasons this CAN'T be an h1. make it a div and put a class on it. then move the style tag to your scss file

Reply:
done